english/LifePuzzle/LifePuzzle.md

```diff
@@ -7,7 +7,7 @@ You are then thrown into an empty glass blender.
 The blades will start moving in 60 seconds. 
 What do you do?
 
-[Ask a question on Quora?](../ask-on-quora/ask-on-quora.md)
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.